packages/vue/src/generators/stories/lib/utils.ts

```diff
@@ -0,0 +1,76 @@
+export function camelCase(input: string): string {
+  if (input.indexOf('-') > 1) {
+    return input
+      .toLowerCase()
+      .replace(/-(.)/g, (_match, group1) => group1.toUpperCase())
+      .replace('.', '');
+  } else {
+    return input;
+  }
+}
+
+export function createDefautPropsObject(propsObject: {
+  [key: string]: string;
+}): {
+  name: string;
+  defaultValue: any;
+}[] {
+  const props = [];
+  for (const key in propsObject) {
+    if (Object.prototype.hasOwnProperty.call(propsObject, key)) {
```

Comment:
Would `for (const key of Object.keys(propsObject))` work better? Just need to enumerable keys from the object right? That way we don't need to check `hasOwnProperty`.